ser574-green-team/taigit
# US187Task245 API to return required data type of task assessment by t…

…he frontend team

US187Task245 based on the communication with frontend team, make a API to return required data type  of task assessment by the frontend team. It helps to reduce API calls.

How to test:
add following codes into taiga.js
import * as taiga from '../libraries/Taiga';
 {console.log(taiga.task_assessment(2577741).then((val) => {console.log('fe', val)} ))}

Expected results:
fe {task_valid: true, finished: true, end_status: "Closed", num_stat: 3, fin_per: 100, …} detail: (3) [{…}, {…}, {…}] end_status: "Closed" fin_per: 100 finished: true num_stat: 3 task_valid: true __proto__: Object

expected detail array:
detail: Array(3) 0: {state_trans_valid: true, timecost: 0, acctime: 0, status_trans: Array(2), user: {…}, …} 1: {state_trans_valid: true, timecost: 762244801, acctime: 762244801, status_trans: Array(2), user: {…}, …} 2: {state_trans_valid: true, timecost: 1628022850, acctime: 1628022850, status_trans: Array(2), user: {…}, …}
